src/encode/util.rs

```diff
@@ -1,37 +1,34 @@
-use std::hash;
 use std::io;
 
 // A Write computing a digest on the bytes written.
-pub struct HasherWrite<'a, W, H>
+pub struct DigestWrite<'a, 'b, W, S>
```

Comment:
Renamed to `DigestWrite` because `Digest` no longer implements `Hash` so we can not provide a generic implementation.